app/(general)/integration/livepeer/page.tsx

```diff
@@ -0,0 +1,20 @@
+'use client'
+
+import { useEffect } from 'react'
+
+import { useRouter } from 'next/navigation'
+
+import { Spinner } from '@/integrations/livepeer/components/spinner'
+
+export default function PageIntegration() {
+  const route = useRouter()
+  useEffect(() => {
+    route.push('/integration/livepeer/vod')
+  }, [])
+
+  return (
+    <div className="flex h-full w-full items-center justify-center">
+      <Spinner />
+    </div>
+  )
+}
```

Comment:
Next provides first class support for [redirect](https://nextjs.org/docs/app/api-reference/functions/redirect) in server components, so you only need to add the `redirect` rather than rendering the page
```suggestion
import { redirect } from 'next/navigation'

export default function PageIntegration() {
  redirect('/integration/livepeer/vod')
}
```